src/main/java/de/codecentric/bootifytestpyramid/web/OrderController.java

```diff
@@ -54,7 +55,7 @@ protected void initBinder(WebDataBinder binder) {
     }
 
     @GetMapping
-    public ResponseEntity<List<OrderResponse>> getAll(@RequestParam(value = "city", required = false) Optional<String> city) {
+    public ResponseEntity<List<OrderResponse>> getAll(@RequestParam("city") Optional<String> city) {
```

Comment:
Oh cool, didn't know you don't need to explicitly set it to `required = false` when using `Optional`